benchmarks/bench_moe_deepseek.py

```
@@ -79,28 +92,44 @@ def is_sm100_family():
+    """Calculate TFLOPS using actual routed token count.
 
-    With EP, only tokens routed to local experts are computed.
-    Assumes uniform routing distribution across experts.
+    FC1: [M, H] x [H, 2I]
+    FC2: [M, I] x [I, H]
+    FLOPs = 2 * local_tokens * (H*2I + I*H) = 6 * local_tokens * H * I
     """
-    if num_local_experts is None:
-        num_local_experts = CFG.num_experts
+    H = CFG.hidden_size
+    I = CFG.intermediate_size
```

Ruff (0.15.5)

[error] Ambiguous variable name: `I`

(E741)